syntax = "proto3";

package protocols.common;

enum GameStage {
  GAME_STAGE_UNSPECIFIED = 0;
  GAME_STAGE_NORMAL_FIRST_HALF_PRE = 1;
  GAME_STAGE_NORMAL_FIRST_HALF = 2;
  GAME_STAGE_NORMAL_HALF_TIME = 3;
  GAME_STAGE_NORMAL_SECOND_HALF_PRE = 4;
  GAME_STAGE_NORMAL_SECOND_HALF = 5;
  GAME_STAGE_EXTRA_TIME_BREAK = 6;
  GAME_STAGE_EXTRA_FIRST_HALF_PRE = 7;
  GAME_STAGE_EXTRA_FIRST_HALF = 8;
  GAME_STAGE_EXTRA_HALF_TIME = 9;
  GAME_STAGE_EXTRA_SECOND_HALF_PRE = 10;
  GAME_STAGE_EXTRA_SECOND_HALF = 11;
  GAME_STAGE_PENALTY_SHOOTOUT_BREAK = 12;
  GAME_STAGE_PENALTY_SHOOTOUT = 13;
  GAME_STAGE_POST_GAME = 14;
}
